client/src/pages/ProfilePage/Education.jsx

```diff
@@ -0,0 +1,187 @@
+import { React, useState } from "react";
+import { Button, Modal } from "flowbite-react";
+import { addEducation } from "./../../api/user";
+import { useDispatch } from "react-redux";
+import { toast } from "sonner";
+import { updateUser } from "./../../state";
+import { useSelector } from "react-redux";
+
+function Education() {
+  const user = useSelector((state) => state.auth.user);
+  const [openModal, setOpenModal] = useState(false);
+  const dispatch = useDispatch();
+  const [school, setSchool] = useState("");
+  const [major, setMajor] = useState("");
+  const [degree, setDegree] = useState("");
+  const [gpa, setGPA] = useState(0);
+  const [start, setStart] = useState(0);
+  const [end, setEnd] = useState(0);
+  const education = user.education;
+
+  const handleSubmit = async (e) => {
+    // e.preventDefault();
+
+    if (!school || !major || !degree) {
+      toast.error("Please fill out all required fields");
+      return;
+    }
+    const newEducation = await addEducation({
+      school: school,
+      major: major,
+      degree: major,
+      gpa: gpa,
+      start_year: start,
+      end_year: end,
+    });
+
+    if (newEducation) {
+      dispatch(updateUser(newEducation));
+      setSchool("");
+      setMajor("");
+      setDegree("");
+      setGPA(0);
+      setStart(0);
+      setEnd(0);
+      setOpenModal(false);
+    }
+  };
+
```

Comment:
you can have a function, resetform() -> use when it create new one, and when they close the modal or sth